src/OSPSuite.Core/Comparison/EventAssignmentBuilderDiffBuilder.cs

```diff
@@ -17,7 +17,7 @@ public override void Compare(IComparison<EventAssignmentBuilder> comparison)
       {
          _entityDiffBuilder.Compare(comparison);
          CompareValues(x => x.UseAsValue, x => x.UseAsValue, comparison);
-         CompareValues(x => x.Dimension, x => x.Dimension, comparison);
+         _objectComparer.Compare(comparison.ChildComparison(x => x.Dimension));
          _objectComparer.Compare(comparison.FormulaComparison());
```

Comment:
I would create an extension methold Like FormulaComparison here
so that you would have 
_objectComparer.Compare(comparison.DimensionComparison());

This would ensure that this is a smaller change, also when implementing in all other apps (MoBi and PKsim)